docs/csharp/language-reference/builtin-types/record.md

```diff
@@ -49,7 +49,7 @@ The preceding examples show some distinctions between records that are reference
 
 The remainder of this article discusses both `record class` and `record struct` types. The differences are detailed in each section. You should decide between a `record class` and a `record struct` similar to deciding between a `class` and a `struct`. The term *record* is used to describe behavior that applies to all record types. Either `record struct` or `record class` is used to describe behavior that applies to only struct or class types, respectively.
 
-## Positional syntax for property definition
+## Positional syntax for property and field definition
 
 You can use positional parameters to declare properties of a record and to initialize the property values when you create an instance:
```

Comment:
Should this intro also mention fields?

Reply:
I didn't on purpose. The default behavior is properties, and fields are introduced a bit later in this section. I can rewrite this one.